src/javascript/_common/base/gtm.js

```diff
@@ -12,35 +12,20 @@ const getAppId       = require('../../config').getAppId;
 const GTM = (() => {
     const isGtmApplicable = () => (/^(1|1098|14473)$/.test(getAppId()));
 
-    const gtmDataLayerInfo = (data) => {
-        const data_layer_info = {
-            language : getLanguage(),
-            pageTitle: pageTitle(),
-            pjax     : State.get('is_loaded_by_pjax'),
-            url      : document.URL,
-            event    : 'page_load',
-        };
-        if (ClientBase.isLoggedIn()) {
-            data_layer_info.visitorId = ClientBase.get('loginid');
-        }
-
-        Object.assign(data_layer_info, data);
-
-        const event = data_layer_info.event;
-        delete data_layer_info.event;
-
-        return {
-            event,
-            data: data_layer_info,
-        };
-    };
+    const getCommonVariables = () => ({
+        language : getLanguage(),
+        pageTitle: pageTitle(),
+        pjax     : State.get('is_loaded_by_pjax'),
+        url      : document.URL,
+        ...ClientBase.isLoggedIn() && { visitorId: ClientBase.get('loginid') },
```

Comment:
we don't need event anymore?

Reply:
it's from `new-app`, we will push event explicitly with `pushDataLayer({ event: 'smth' } )`

Reply:
forgot to pass `page_load` though in binary_loader 😅